libs/VideoKit/source/VideoKit.cpp

```diff
@@ -29,14 +29,24 @@ void VideoKit::initializeScreen()
 	_thread.start(mbed::Callback(this, &VideoKit::run));
 }
 
-void VideoKit::displayImage(const std::filesystem::path &path)
+void VideoKit::displayImage(const std::filesystem::path &path, bool display_white_backgroung)
 {
 	const std::scoped_lock lock(mutex);
 
 	if (path == _current_path) {
 		return;
 	}
 
+	if (display_white_backgroung) {
+		_event_flags.set(flags::STOP_VIDEO_FLAG);
+		auto file = FileManagerKit::File {"fs/home/img/system/robot-misc-robot-misc-screen_empty_white.jpg"};
+
+		rtos::ThisThread::sleep_for(100ms);
+
+		_video.displayImage(file);
+		file.close();
+	}
```

Comment:
Must arrived after `_event_flags.set(flags::STOP_VIDEO_FLAG);`, otherwise the video will play, flash and stop with something different of a _empty white screen_.

Also let a delay before display it, 'cause of time for video thread to stop after the raise of STOP_VIDEO_FLAG